README.md

```diff
@@ -400,9 +400,9 @@ var web3 = new Web3(new Web3.providers.HttpProvider("http://localhost:8545"));
 // NOTE: if you run Kovan node there should be an address you've got in the "Option 2: Run Kovan node" step
 web3.eth.defaultAccount = "0x004ec07d2329997267ec62b4166639513386f32e";
 // read JSON ABI
-var abi = JSON.parse(fs.readFileSync("./target/json/TokenContract.json"));
+var abi = JSON.parse(fs.readFileSync("./target/json/TokenInterface.json"));
 // convert Wasm binary to hex format
-var codeHex = '0x' + fs.readFileSync("./target/pwasm_tutorial_contract.wasm").toString('hex');
+var codeHex = '0x' + fs.readFileSync("./target/wasm32-unknown-unknown/release/pwasm_tutorial_contract.wasm").toString('hex');
```

Comment:
It's important to understand that `./target/wasm32-unknown-unknown/release/pwasm_tutorial_contract.wasm` is the raw contract code, generated by cargo. It's not suitable for deployment because it's not a constructor. In order to build constructor we take this file and pass it through `wasm-build` tool (https://github.com/paritytech/pwasm-tutorial#parity-wasm-build).
It seems like the https://github.com/paritytech/pwasm-tutorial#building part should be more descriptive in respect to the https://github.com/paritytech/pwasm-tutorial/blob/master/step-0/build.sh contents..

Reply:
Oh, damn, you've got me on this.
Is it possible to extend `build.rs` to do the `wasm-build` magic automatically?

Reply:
no way. `build.rs` run before cargo build. But we need to run `wasm-build` when we have the binary